### ISSUE TITLE
Failed to fetch sample 126973

### ISSUE THREAD

**Issue by CarlHuangNuc** [xlang-ai/aguvis#2]
use_mps_device=False,
verbose_logging=False,
warmup_ratio=0.03,
warmup_steps=0,
weight_decay=0.0,
)

Rank 0:  Loading data/aguvis/train/ricosca.json with all sampling strategy
Rank 0:  Loaded 173212 samples from data/aguvis/train/ricosca.json
Rank 0:  Loaded 173212 samples from data/stage1.yaml
Rank 0:  Formatting inputs...Skip in lazy mode
/home/test/data/aguvis/train.py:193: FutureWarning: `tokenizer` is deprecated and will be removed in version 5.0.0 for `AGUVISTrainer.__init__`. Use `processing_class` instead.
  trainer = AGUVISTrainer(
/home/test/data/aguvis/train.py:193: FutureWarning: `tokenizer` is deprecated and will be removed in version 5.0.0 for `AGUVISTrainer.__init__`. Use `processing_class` instead.
  trainer = AGUVISTrainer(
/home/test/data/aguvis/train.py:193: FutureWarning: `tokenizer` is deprecated and will be removed in version 5.0.0 for `AGUVISTrainer.__init__`. Use `processing_class` instead.
  trainer = AGUVISTrainer(
/home/test/data/aguvis/train.py:193: FutureWarning: `tokenizer` is deprecated and will be removed in version 5.0.0 for `AGUVISTrainer.__init__`. Use `processing_class` instead.
  trainer = AGUVISTrainer(
/home/test/data/aguvis/train.py:193: FutureWarning: `tokenizer` is deprecated and will be removed in version 5.0.0 for `AGUVISTrainer.__init__`. Use `processing_class` instead.
  trainer = AGUVISTrainer(
/home/test/data/aguvis/train.py:193: FutureWarning: `tokenizer` is deprecated and will be removed in version 5.0.0 for `AGUVISTrainer.__init__`. Use `processing_class` instead.
  trainer = AGUVISTrainer(
/home/test/data/aguvis/train.py:193: FutureWarning: `tokenizer` is deprecated and will be removed in version 5.0.0 for `AGUVISTrainer.__init__`. Use `processing_class` instead.
  trainer = AGUVISTrainer(
/home/test/data/aguvis/train.py:193: FutureWarning: `tokenizer` is deprecated and will be removed in version 5.0.0 for `AGUVISTrainer.__init__`. Use `processing_class` instead.
  trainer = AGUVISTrainer(
Parameter Offload: Total persistent parameters: 877056 in 401 params
wandb: WARNING The `run_name` is currently set to the same value as `TrainingArguments.output_dir`. If this was not intended, please specify a different run name by setting the `TrainingArguments.run_name` parameter.
wandb: Using wandb-core as the SDK backend. Please refer to https://wandb.me/wandb-core for more information.
wandb: (1) Create a W&B account
wandb: (2) Use an existing W&B account
wandb: (3) Don't visualize my results
wandb: Enter your choice: 3
wandb: You chose "Don't visualize my results"
wandb: Tracking run with wandb version 0.18.3
wandb: W&B syncing is set to `offline` in this directory.
wandb: Run `wandb online` or set WANDB_MODE=online to enable cloud syncing.
  0%|                                                                                                                                                                                                                                                               | 0/10826 [00:00<?, ?it/s]Failed to fetch sample 170741. Exception: Unable to create tensor, you should probably activate padding with 'padding=True' to have batched tensors with the same length.
Failed to fetch sample 126973. Exception: Unable to create tensor, you should probably activate padding with 'padding=True' to have batched tensors with the same length.
Failed to fetch sample 17238. Exception:Failed to fetch sample 134260. Exception: Unable to create tensor, you should probably activate padding with 'padding=True' to have batched tensors with the same length. Unable to create tensor, you should probably activate padding with 'padding=True' to have batched tensors with the same length.
